va/va.h

```diff
@@ -1810,6 +1810,45 @@ vaQueryProcessingRate(
     unsigned int       *processing_rate
 );
 
+/**
+ * \brief Device info structure
+ */
+typedef struct _VADeviceInfo {
+    /** \brief Device id */
+    int32_t device_id;
+    /** \brief Vendor id */
+    int32_t vendor_id;
```

Comment:
Per my taste `vendor_id` should go before `device_id`. Please, reorder.

Reply:
Done